src/main/scala/org/broadinstitute/dsde/workbench/leonardo/model/LeonardoModel.scala

```diff
@@ -37,6 +37,7 @@ final case class ClusterRequest(labels: LabelMap,
                           defaultClientId: Option[String] = None,
                           jupyterDockerImage: Option[String] = None,
                           rstudioDockerImage: Option[String] = None,
+                          welderDockerImage: Option[String] = None,
```

Comment:
I would have preferred to restructure this a la

```
{
    ...
    "dockerConfiguration": {
        "jupyter": "us.gcr.io/broad/jupyter:fakehash",
        "welder": "us.gcr.io/broad/welder:fakehash"
    }
    ...
}``

but this would have required creating a new endpoint version and it did not seem worth it to do that at this time.